Spigot-Server-Patches/0592-Add-PlayerShearBlockEvent.patch

```diff
@@ -0,0 +1,94 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: JRoy <joshroy126@gmail.com>
+Date: Thu, 27 Aug 2020 15:02:48 -0400
+Subject: [PATCH] Add PlayerShearBlockEvent
+
+
+diff --git a/src/main/java/net/minecraft/server/BlockBeehive.java b/src/main/java/net/minecraft/server/BlockBeehive.java
+index 7e2c63e4731ac2d234d5f90eb80d314cdede07ca..b95c602e89aa1dd55b1c76cce26d48943abf4cff 100644
+--- a/src/main/java/net/minecraft/server/BlockBeehive.java
++++ b/src/main/java/net/minecraft/server/BlockBeehive.java
+@@ -1,5 +1,7 @@
+ package net.minecraft.server;
+ 
++import io.papermc.paper.event.block.PlayerShearBlockEvent; // Paper - PlayerShearBlockEvent namespace conflicts
++
+ import java.util.Iterator;
+ import java.util.List;
+ import java.util.Random;
+@@ -63,7 +65,13 @@ public class BlockBeehive extends BlockTileEntity {
+     }
+ 
+     public static void a(World world, BlockPosition blockposition) {
+-        a(world, blockposition, new ItemStack(Items.HONEYCOMB, 3));
```

Comment:
Please keep the OBF helper to the first example right above it, 
https://github.com/PaperMC/Paper/blob/master/CONTRIBUTING.md#obfuscation-helpers
This just creates extra noisy diffs

Reply:
Looking back at this, this was the "wrong" change, if you just want an OBFHELPER, it should be added to Block, so that that `a` method is mapped (ideally in MCUtils for later calls) for anything that wants to call it, as this is not so much an obfhelper here but a behavioral change to the method in the name of an obfhelper, beyond that, looks ok